main.js

```diff
@@ -0,0 +1,35 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+    let allNumbers = [];
+
+    for (i = 0; i < numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 101);
+
+        allNumbers.push(randNumber);
+    }
+
+    return allNumbers;
+}
+
+
+function displayNumbers(allNumbers) {
+    let numbersOnScreen = allNumbers.length;
+
+    for (i = 0; i < numbersOnScreen; i++) {
```

Comment:
why use a `for` here and not a `forEach`? then you wouldnt need to get the length